src/main/java/seedu/duke/logic/parser/Parser.java

```diff
@@ -11,6 +11,10 @@
 //@@author richwill28
 public class Parser {
     public static Command parse(String userResponse) throws ParseException {
+        if (userResponse.contains("|")) {

```

Comment:
Perhaps instead of putting this here, you could put it in the `AddCommandParser` class? Since it only really affects the adding of items for now.